data/gahuza/legacyAssets/video/2016/01/160108_australia_fire_video.json

```diff
@@ -22,7 +22,8 @@
     },
     "analyticsLabels": {
       "cps_asset_type": "map",
-      "cps_asset_id": "42074194"
+      "cps_asset_id": "42074194",
```

Comment:
I know this hasn't been changed, and it's only fixture data... but this appears to be a CPS Asset ID in a TopCat asset, which I don't think would ever be a thing in reality.

Reply:
Fixture seems to match the asset from live.

Reply:
So it does.  Okay, no worries then :)